homeassistant/components/drop/strings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "abort": {
+      "not_supported": "Configuration for DROP is through MQTT discovery.  Use the DROP Connect app to connect your DROP Hub to your MQTT broker."
+    },
+    "step": {
+      "confirm": {
+        "title": "Confirm Association",
+        "description": "Do you want to use the DROP {device_type} named {device_name}?'"
```

Comment:
```suggestion
        "description": "Do you want to configure the DROP {device_type} named {device_name}?'"
```

Reply:
I agree this is more clear.  Changed in 2ad2977100239bde557e0203a206667b39b621c8.